contracts/SportMarkets/SportsAMM.sol

```diff
@@ -545,14 +552,27 @@ contract SportsAMM is Initializable, ProxyOwned, PausableUpgradeable, ProxyReent
     ) internal {
         require(isMarketInAMMTrading(market), "Not in Trading");
         require(ISportPositionalMarket(market).optionsCount() > uint(position), "Invalid position");
+
+        uint baseOdds = obtainOdds(market, position);
+
+        require(baseOdds > 0, "No base odds");
+
+        baseOdds = baseOdds < minSupportedOdds ? minSupportedOdds : baseOdds;
+
         uint availableToBuyFromAMMatm = availableToBuyFromAMM(market, position);
         require(amount > ZERO_POINT_ONE && amount <= availableToBuyFromAMMatm, "Low liquidity || 0 amount");
 
         if (sendSUSD) {
             if (msg.sender == parlayAMM) {
-                sUSDPaid = buyFromAmmQuoteForParlayAMM(market, position, amount);
```

Comment:
thought it might break the ParlayAMM, but it's just the use of internal function